### ISSUE TITLE
cura 5.3 v400 support error

### ISSUE THREAD

**Issue by sadamop** [Ultimaker/Cura#15326]
### Application Version

5.3

### Platform

windows 10 home 

### Printer

flsun v400

### Reproduction steps

when generating a support does not start to the first layer, but part to the second layer consequently does not go
![1 layer](https://user-images.githubusercontent.com/131551829/233801432-41dce122-4adf-413c-a33f-6d87aa7e9bc0.PNG)
![2 layer](https://user-images.githubusercontent.com/131551829/233801438-9a068697-15fe-4421-a20e-e64741966171.PNG)

### Actual results

nothing because the supports so you can not use

### Expected results

will solve the problem soon

### Checklist of files to include

- [ ] Log file
- [ ] Project file

### Additional information & file uploads

nothing to add

**Comment by GregValiant** [comment]
Thanks for the report.
We really need a project file to investigate this.  Load the model and use the "File | Save Project" command.  Zip the file and post the zip folder here.

**Comment by sadamop** [comment]
Hi the problem is not the file because in cura 5.2.2 this generate correctly the support the problem is all cura 5.3 i have much friends 
with the same problem on all files and unfortunately we have to use version 5.2.2 but there is no flsun v400 profile and we have problems
Inviato da Libero Mail

**Comment by casperlamboo** [comment]
Hi @sadamop,

The problem is indeed most likely in the program. However for us to properly debug the issue we need the file in order to properly see what is going on, and to see if the fix will actually resolve this problem.

To create the project file click safe from the file menu (or press `ctrl+s`). The create a zip from the exported 3mf file and share this zip on this page on github. Once this is done we can properly resolve the issue.

Thank you in advance.

**Comment by sadamop** [comment]
[problem support.zip](https://github.com/Ultimaker/Cura/files/11330887/problem.support.zip)

**Comment by GregValiant** [comment]
Thanks for the file.
@casperlamboo I can duplicate this when the Bed Adhesion is set to "None" and the "Enable Support Brim" is checked.
@sadamop A temporary fix is to set the Bed Adhesion to Skirt or Brim and/or to disable the Support Brim.

**Comment by sadamop** [comment]
ok tnx 
ok now I saw, if you do not use any adhesion to the plate the support does not go to the first layer.
 but if you set brim or skirt to the first true layer generated with skirt or brim and the true support above, but I hope you solve this problem

**Comment by MariMakes** [comment]
Hey All,

This seems to be a duplicate of https://github.com/Ultimaker/Cura/issues/14983
We have a ticket on our backlog with the intent to improve this behavior.

I'll be closing this issue as a duplicate but you can follow the progress here: https://github.com/Ultimaker/Cura/issues/14983
Thanks for your report 👍
